contributing/samples.md

```diff
@@ -0,0 +1,258 @@
+## Creating a new sample
+
+Samples live in `examples/source`. Pick one of the existing category folders. 
+
+If you sample can live in a single html file, create a new `*.html` in  one of the sample category folders:
+
+```shell
+$ vim examples/source/1.components/amp-awesome.html
+```
+
+If your sample requires a custom backend API, create a new folder which is going to contain both, your sample html file, which must be named `index.html` and your api implementation `api.js`.
+
+```shell
+$ mkdir examples/source/1.components/amp-awesome
+$ vim examples/source/1.components/amp-awesome/index.html
+$ vim examples/source/1.components/amp-awesome/api.js
+```
+
+If your sample, does not fit into one of the existing categories, please [create an issue](https://github.com/ampproject/docs/issues/new) first and ask for feedback.
```

Comment:
Should we encourage to ping the team either on #wg-outreach or via an issue in any case? This would enable us to push example ideas in a specific direction to cover certain use cases.

Reply:
I think a formal issue is better in this case